src/server/pfs/s3/s3.go

```diff
@@ -64,6 +64,10 @@ func (c *controller) requestClient(r *http.Request) *client.APIClient {
 	vars := mux.Vars(r)
 	if vars["s3gAuth"] != "disabled" {
 		accessKey := vars["authAccessKey"]
+		if strings.HasPrefix(accessKey, "PAC1") {
+			vars["isProjectAware"] = "yes"
```

Comment:
does the `isProjectAware` param get used anywhere?

Reply:
nvm 👍 read over CORE-1116

Reply:
Yup, it gets set here and in [auth.go](https://github.com/pachyderm/pachyderm/blob/f7f5088b2f5f2a43722c49f09e8b5a9ff540376a/src/server/pfs/s3/auth.go#L20), and it gets used in [listBuckets](https://github.com/pachyderm/pachyderm/blob/f7f5088b2f5f2a43722c49f09e8b5a9ff540376a/src/server/pfs/s3/driver.go#L71) and [bucket](https://github.com/pachyderm/pachyderm/blob/f7f5088b2f5f2a43722c49f09e8b5a9ff540376a/src/server/pfs/s3/driver.go#L163).

It would be nice to only set it in one place, but that would require a change in the s2 package.